draft-ietf-rtcweb-jsep.xml

```diff
@@ -2365,7 +2365,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <t>The m= line and corresponding "a=rtpmap" and "a=fmtp"
             lines MUST only include codecs present in the most recent
             answer which have not been excluded by the codec preferences
-            of the associated transceiver.</t>
+            of the associated transceiver. Note that non-JSEP endpoints
+            are not subject to these restrictions, and MAY offer media
```

Comment:
I would not make this a capitalized MAY because this is informative not normative.

One way to handle this is to say ", and therefore JSEP endpoints MUST be prepared to receive..."